src/syscheckd/create_db.c

```diff
@@ -46,7 +46,131 @@ static const char *FIM_EVENT_MODE[] = {
     "whodata"
 };
 
-// Callback
+typedef struct fim_txn_context_s {
```

Comment:
Is a structure necessary to contain a single variable?

Reply:
Maybe in a future we need to add more variable to the context of a transaction, this will avoid changes in the function headers